New Cppcheck errors: '[unreadVariable]'

**Checklist**
* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Latest version of Cppcheck is issuing the following failures for HS:
```
fsw/src/hs_cmds.c:157:23: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32  Status     = CFE_SUCCESS;
                      ^
fsw/src/hs_cmds.c:158:23: style: Variable 'TableIndex' is assigned a value that is never used. [unreadVariable]
    uint32 TableIndex = 0;
                      ^
fsw/src/hs_custom.c:267:22: style: Variable 'j' is assigned a value that is never used. [unreadVariable]
    uint32 j         = 0;
                     ^
fsw/src/hs_custom.c:268:22: style: Variable 'ThisValue' is assigned a value that is never used. [unreadVariable]
    uint32 ThisValue = 0;
                     ^
fsw/src/hs_custom.c:270:28: style: Variable 'Ordinal' is assigned a value that is never used. [unreadVariable]
    uint32 Ordinal         = 0;
                           ^
fsw/src/hs_custom.c:271:28: style: Variable 'NewOrdinalIndex' is assigned a value that is never used. [unreadVariable]
    uint32 NewOrdinalIndex = 0;
                           ^
fsw/src/hs_monitors.c:48:35: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32            Status       = CFE_SUCCESS;
                                  ^
fsw/src/hs_monitors.c:50:35: style: Variable 'ActionType' is assigned a value that is never used. [unreadVariable]
    uint16           ActionType   = 0;
                                  ^
fsw/src/hs_monitors.c:228:35: style: Variable 'ActionType' is assigned a value that is never used. [unreadVariable]
    uint16           ActionType   = 0;
                                  ^
fsw/src/hs_monitors.c:458:24: style: Variable 'EntryResult' is assigned a value that is never used. [unreadVariable]
    int32  EntryResult = 0;
                       ^
fsw/src/hs_monitors.c:460:23: style: Variable 'ActionType' is assigned a value that is never used. [unreadVariable]
    uint16 ActionType = 0;
                      ^
fsw/src/hs_monitors.c:461:23: style: Variable 'CycleCount' is assigned a value that is never used. [unreadVariable]
    uint16 CycleCount = 0;
                      ^
fsw/src/hs_monitors.c:462:23: style: Variable 'NullTerm' is assigned a value that is never used. [unreadVariable]
    uint16 NullTerm   = 0;
                      ^
fsw/src/hs_monitors.c:549:24: style: Variable 'EntryResult' is assigned a value that is never used. [unreadVariable]
    int32  EntryResult = 0;
                       ^
fsw/src/hs_monitors.c:551:23: style: Variable 'ActionType' is assigned a value that is never used. [unreadVariable]
    uint16 ActionType = 0;
                      ^
fsw/src/hs_monitors.c:552:23: style: Variable 'EventID' is assigned a value that is never used. [unreadVariable]
    uint16 EventID    = 0;
                      ^
fsw/src/hs_monitors.c:553:23: style: Variable 'NullTerm' is assigned a value that is never used. [unreadVariable]
    uint16 NullTerm   = 0;
                      ^
fsw/src/hs_monitors.c:734:24: style: Variable 'EnableState' is assigned a value that is never used. [unreadVariable]
    uint16 EnableState = 0;
                       ^
fsw/src/hs_monitors.c:735:24: style: Variable 'EntryResult' is assigned a value that is never used. [unreadVariable]
    int32  EntryResult = 0;
                       ^
##[error]Process completed with exit code 255.
```

**To Reproduce**
Run the current version of Cppcheck on the current main branch HS source code.

**Expected behavior**
Cppcheck should pass without raising any errors.

**Reporter Info**
Avi @thnkslprpt

--- a/fsw/src/hs_cmds.c
+++ b/fsw/src/hs_cmds.c
@@ -154,8 +154,8 @@ void HS_HousekeepingReq(const CFE_SB_Buffer_t *BufPtr)
     CFE_ES_TaskInfo_t  TaskInfo;
     memset(&TaskInfo, 0, sizeof(TaskInfo));
 #endif
-    int32  Status     = CFE_SUCCESS;
-    uint32 TableIndex = 0;
+    int32  Status;
+    uint32 TableIndex;
 
     /*
     ** Verify message packet length

--- a/fsw/src/hs_custom.c
+++ b/fsw/src/hs_custom.c
@@ -263,12 +263,12 @@ void HS_UtilDiagReport(void)
 {
     uint32 DiagValue[HS_UTIL_TIME_DIAG_ARRAY_LENGTH];
     uint32 DiagCount[HS_UTIL_TIME_DIAG_ARRAY_LENGTH];
-    uint32 i         = 0;
-    uint32 j         = 0;
-    uint32 ThisValue = 0;
+    uint32 i = 0;
+    uint32 j;
+    uint32 ThisValue;
 
-    uint32 Ordinal         = 0;
-    uint32 NewOrdinalIndex = 0;
+    uint32 Ordinal;
+    uint32 NewOrdinalIndex;
     uint32 OutputValue[HS_UTIL_DIAG_REPORTS];
     uint32 OutputCount[HS_UTIL_DIAG_REPORTS];
     uint32 OutputOrdinal[HS_UTIL_DIAG_REPORTS];

--- a/fsw/src/hs_monitors.c
+++ b/fsw/src/hs_monitors.c
@@ -44,10 +44,10 @@
 void HS_MonitorApplications(void)
 {
     CFE_ES_AppInfo_t AppInfo;
-    CFE_ES_AppId_t   AppId        = CFE_ES_APPID_UNDEFINED;
-    int32            Status       = CFE_SUCCESS;
-    uint32           TableIndex   = 0;
-    uint16           ActionType   = 0;
+    CFE_ES_AppId_t   AppId = CFE_ES_APPID_UNDEFINED;
+    int32            Status;
+    uint32           TableIndex = 0;
+    uint16           ActionType;
     uint32           MsgActsIndex = 0;
     CFE_SB_Buffer_t *BufPtr       = NULL;
 
@@ -222,10 +222,10 @@ void HS_MonitorApplications(void)
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 void HS_MonitorEvent(const CFE_EVS_LongEventTlm_t *EventPtr)
 {
-    uint32           TableIndex   = 0;
-    int32            Status       = CFE_SUCCESS;
-    CFE_ES_AppId_t   AppId        = CFE_ES_APPID_UNDEFINED;
-    uint16           ActionType   = 0;
+    uint32           TableIndex = 0;
+    int32            Status     = CFE_SUCCESS;
+    CFE_ES_AppId_t   AppId      = CFE_ES_APPID_UNDEFINED;
+    uint16           ActionType;
     uint32           MsgActsIndex = 0;
     CFE_SB_Buffer_t *SendPtr      = NULL;
 
@@ -455,11 +455,11 @@ int32 HS_ValidateAMTable(void *TableData)
 
     int32  TableResult = CFE_SUCCESS;
     uint32 TableIndex  = 0;
-    int32  EntryResult = 0;
+    int32  EntryResult;
 
-    uint16 ActionType = 0;
-    uint16 CycleCount = 0;
-    uint16 NullTerm   = 0;
+    uint16 ActionType;
+    uint16 CycleCount;
+    uint16 NullTerm;
 
     uint32 GoodCount                = 0;
     uint32 BadCount                 = 0;
@@ -546,11 +546,11 @@ int32 HS_ValidateEMTable(void *TableData)
 
     int32  TableResult = CFE_SUCCESS;
     uint32 TableIndex  = 0;
-    int32  EntryResult = 0;
+    int32  EntryResult;
 
-    uint16 ActionType = 0;
-    uint16 EventID    = 0;
-    uint16 NullTerm   = 0;
+    uint16 ActionType;
+    uint16 EventID;
+    uint16 NullTerm;
 
     uint32 GoodCount                = 0;
     uint32 BadCount                 = 0;
@@ -731,8 +731,8 @@ int32 HS_ValidateMATable(void *TableData)
     int32  TableResult = CFE_SUCCESS;
     uint32 TableIndex  = 0;
     size_t Length      = 0;
-    uint16 EnableState = 0;
-    int32  EntryResult = 0;
+    uint16 EnableState;
+    int32  EntryResult;
 
     CFE_SB_MsgId_t   MessageID = CFE_SB_INVALID_MSG_ID;
     CFE_SB_Buffer_t *BufPtr    = (CFE_SB_Buffer_t *)NULL;

